client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -110,24 +108,19 @@ export class List extends React.Component {
 		const { translate } = this.props;
 
 		return (
-			<Notice
-				status="is-success"
-				showDismiss={ false }
-				text={ translate( 'Free domain available' ) }
-				icon="info-outline"
+			<UpsellNudge
+				callToAction={ translate( 'Claim Free Domain' ) }
+				title={ translate( 'Free domain available' ) }
 				className="domain-management__claim-free-domain"
-			>
-				<NoticeAction
-					onClick={ this.props.clickClaimDomainNotice }
-					href={ `/domains/add/${ this.props.selectedSite.slug }` }
-				>
-					{ translate( 'Claim Free Domain' ) }
-					<TrackComponentView
-						eventName={ 'calypso_domain_credit_reminder_impression' }
-						eventProperties={ { cta_name: 'domain_info_notice' } }
-					/>
-				</NoticeAction>
-			</Notice>
+				forceDisplay
+				href={ `/domains/add/${ this.props.selectedSite.slug }` }
```

Comment:
It's odd to me to see some props destructured (`translate`) but not others, although it's easy to argue `hasDomainCredit` and `selectedSite` aren't used more than once – whichever you think is best 🙃

Reply:
It looks like `{ hasDomainCredit }` is already declared as a selector, so it throws an error when you destructure it...it also looks like the function returns null when `hasDomainCredit` is false, so it never gets to the other two. I destructured `selectedSite` for consistency with `translate`, though.